server/core/src/main/java/com/mongodb/stitch/server/core/auth/internal/StitchAuthImpl.java

```diff
@@ -90,6 +92,42 @@ public void removeUserWithId(final String userId) {
     removeUserWithIdInternal(userId);
   }
 
+  @Override
+  protected void onListenerInitialized() {
```

Comment:
[q] why do we need to implement these? shouldn't there already be default implementations in the protocol?

Reply:
These are abstract methods on CoreStitchAuth to telegraph to the implementor that it must pass on these methods.